neukinnis/common/models/account.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "account",
+  "base": "PersistedModel",
+  "idInjection": true,
+  "properties": {
+    "name": {
+      "type": "string",
+      "required": true
+    },
+    "account_type": {
```

Comment:
This is what I was stuck on. I wanted federal, commercial, and enterprise to be the only valid account_type. Is this the proper way for the API to reject any non-valid values? I was trying to find a way in the json model schema to fix those account_type. 
`    "account_type": {
      "type": "string",
      "required": true,
      "strict": true,
      "enum": ["Federal", "Enterprise", "Commercial"]              }
  }`

I thought enum might work based on research. Does this not work because it's going through loopback?